apparmor.d/groups/freedesktop/polkitd

```diff
@@ -22,14 +22,30 @@ profile polkitd @{exec_path} {
 
   ptrace (read),
 
-  dbus (send,receive) bus=system path=/org/freedesktop/PolicyKit[0-9]/*
-       interface=org.freedesktop.{DBus.Introspectable,DBus.Properties,PolicyKit[0-9].*},
-
-  dbus send bus=system path=/org/freedesktop/DBus
+  dbus (send) bus=system path=/org/freedesktop/DBus
```

Comment:
The idea here is to granularize and allow sending all members, but receive only mentioned.

Reply:
That's look like a pain to manage all of this for all profiles.

I mean, in concept `polkitd` 'own' the `/org/freedesktop/PolicyKit[0-9]` namespace in dBus. Therefore it makes sense to allow everything in it regardless send or receive. (cf the systemd polkit service definition that does the same).

It is also a simplification as a change in the program would not change the profile.

Reply:
Alright, that's reasonable. I hope send/receive parenthesis is OK.

Reply:
As you might have seen, I usually do not put them unless it is mandatory, but honestly we have other priorities than tracking parenthesis.